src/components/modules/dashboard/ui/cards/EscrowSection.tsx

```
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tab";
+import { StatusBreakdownChart } from "./StatusBreakdownChart";
+import { StatusBreakdownTable } from "./StatusBreakdownTable";
+import { TopEscrowsTable } from "./TopEscrowsTable";
+import { EscrowTrendsChart } from "./EscrowTrendsChart";
+import { EscrowVolumeChart } from "./EscrowVolumnChart";
+import { useEscrowDashboardSection } from "@/hooks/use-escrow-section.hook";
+
+export function EscrowSection() {
+  const [useMockData, setUseMockData] = useState(true);
```

ESLint

[error] 'setUseMockData' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)